idaes/core/util/model_diagnostics.py

```diff
@@ -694,10 +713,152 @@ def find_irreducible_degenerate_sets(self, verbose=True, tee=False):
     # Note: This makes sense as a static method
     @staticmethod
     def print_variable_bounds(v):
-        """Print variable, bounds, and value
+        """
+        Print variable, bounds, and value
+
         Argument:
             v: variable
+
         Return:
             nothing
```

Comment:
Consider changing `Argument` to `Arguments` and `Return` to `Returns`? Docstring not rendered as expected in online pages.